src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,292 @@
+package scala
+package collection.mutable
```

Comment:
Hi, indeed we should put new collections into `strawman.collection`. That will make it easier to have both the old and new collections at the same time, so that we can compare them.